operations/commit_queue.go

```diff
@@ -319,6 +319,9 @@ func (p *mergeParams) uploadMergePatch(conf *ClientSettings, ac *legacyClient) e
 		}
 		return errors.Wrap(err, "can't get project ref")
 	}
+	if !ref.CommitQueue.Enabled && ref.CommitQueue.PatchType == commitqueue.CLIPatchType {
```

Comment:
I think we would want to check `!(ref.CommitQueue.Enabled && ref.CommitQueue.PatchType == commitqueue.CLIPatchType)` or the equivalent.

Reply:
Sorry, apparently I don't pay close enough attention when I juggle tickets :)